GoogleChromeLabs/wp-sitemaps
# Feature/80 add homepage

### Issue Number
https://github.com/GoogleChromeLabs/wp-sitemaps/issues/80 

### Description
If the homepage has been set to show latest posts, then the main homepage URL displays on the pages sitemap with a lastmod date of the latest post on the site. 

This isn't needed if set to display a static page as that page is included within the pages sitemap automatically, and displays the lastmod date for that page.

### Screenshots (before and after if applicable)

**Your homepage displays latest posts**

<img width="751" alt="Screenshot 2019-11-22 at 15 58 13" src="https://user-images.githubusercontent.com/182015/69440574-ff44fa00-0d40-11ea-8f4b-09e3f7e62d43.png">

**Your homepage displays a static page**

<img width="751" alt="Screenshot 2019-11-22 at 15 58 35" src="https://user-images.githubusercontent.com/182015/69440564-fbb17300-0d40-11ea-8716-19dda9702808.png">

### Type of change
Please select the relevant options:


### Steps to test
- View the sitemap index locally
- Log in to the admin in another tab and go to Settings > Reading.
- Set your WordPress install to "your homepage displays latest posts" to see the home url display on the sitemap index.

### Acceptance criteria
